spec/requests/api/v1/users/logout_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe 'Logout User', type: :request do
+  subject { delete '/api/v1/auth/sign_out', headers:, as: :json }
+
+  before(:each) do
+    @current_user = FactoryBot.create(:user)
+    @headers = @current_user.create_new_auth_token
+  end
+
+  context 'Correctly logout the user' do
```

Comment:
```suggestion
  context 'when success' do
```
you already define de description of the test as `Log Out` user, so when it raise an error because the test fails it will said `Log Out user when correctly log out user`.. it's a bit repetitive